Classes/KIFTestActor.m

```diff
@@ -179,7 +180,13 @@ - (void)waitForTimeInterval:(NSTimeInterval)timeInterval
 }
 
 -(void)waitForAnimationsToFinish {
-    NSTimeInterval maximumWaitingTimeInterval = 3;
+    NSTimeInterval maximumWaitingTimeInterval = self.animationWaitingTimeout;
+    if(maximumWaitingTimeInterval <= 0) {
+        return;
+    }
+    
+    // Wait for the view to stabilize and give them a chance to start animations before we wait for them.
+    [self waitForTimeInterval:0.5f];
     
     NSTimeInterval startTime = [NSDate timeIntervalSinceReferenceDate];
     [self runBlock:^KIFTestStepResult(NSError **error) {
```

Comment:
Rather than using `runBlock:timeout:` and passing a timeInterval + 1 second to prevent the exception from being raised, it seems like it would make more sense to use `tryRunningBlock:complete:timeout:error:`. That way, you don't need your own time interval checking on line 209.